index.html

```diff
@@ -298,6 +306,22 @@
             app.appendChild(scoringSection);
         }
 
+        function playAlarmSound() {
+            const context = new (window.AudioContext || window.webkitAudioContext)();
+            const oscillator = context.createOscillator();
+            const gainNode = context.createGain();
+
+            oscillator.connect(gainNode);
+            gainNode.connect(context.destination);
+
+            gainNode.gain.value = 0.5;
+            oscillator.frequency.value = 440;
+            oscillator.type = "sine";
+
+            oscillator.start(context.currentTime);
+            oscillator.stop(context.currentTime + 0.5);
+        }
+
```

Comment:
To add an alarm sound when the timer completes, we'll use the Web Audio API. We'll create a function to play a simple beep sound and call it when the timer reaches zero. Here's the updated code: